packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -436,7 +451,7 @@ export const TooltipPlugin2 = ({ config, hoverMode, render, clientZoom = false,
   return null;
 };
 
-const getStyles = (theme: GrafanaTheme2) => ({
+const getStyles = (theme: GrafanaTheme2, maxWidth: number | undefined, maxHeight: number | undefined) => ({
```

Comment:
Will `maxWidth` and `maxHeight` ever be "undefined", since we give a default value above?

Reply:
Good catch, thanks!